### PR TITLE
Added images, title and description files

### PR DESCRIPTION
**Pull request by nickanders**

### CODE REVIEW

File: e-conomic_to_bullsender/description.md [hunk 1]
```diff
@@ -0,0 +1 @@
+Add your existing and new contacts in e-conomic to your Bullsender account
```
Comment (zubairov) [inline e-conomic_to_bullsender/description.md:1]: suggestion here 

> ...new contacts **from** e-conomic to your ...

File: debitoor_to_bullsender/description.md [hunk 2]
```diff
@@ -0,0 +1 @@
+Add your existing and new contacts Debitoor to your Bullsender account
```
Comment (zubairov) [inline debitoor_to_bullsender/description.md:1]: suggestion for the name:

> ..new contacts **from** Debitoor to your...

**Comment by zubairov** [comment]
@pauseapps please say 'yes' to my comments and I'll merge it :)

**Comment by nickanders** [comment]
Yes. Sounds great, let's do it :)

**Pull request merged**